war/src/main/js/add-item.js

```diff
@@ -116,6 +116,18 @@ $.when(getItems()).done(function(data) {
       }
     }
 
+    function focusButton(status) {
+      var btn = $('form .footer .btn-decorator button[type=submit]');
+      if (status === true) {
+        btn.focus();
+      }
```

Comment:
`else` functionality was not present initially and seems that there may not be much need for it. Or do you think it would be better with `else` ?

Ok, I moved selecting inside if block.